pkg/controller/machinehealthcheck/machinehealthcheck_controller_test.go

```diff
@@ -1830,6 +1830,59 @@ func TestNeedsRemediation(t *testing.T) {
 			expectedNextCheck:           time.Duration(1 * time.Minute), // 300-200 rounded
 			expectedError:               false,
 		},
+		{
+			testCase: "unhealthy: machine reached its maximal age",
+			target: &target{
+				Machine: mapiv1beta1.Machine{
+					TypeMeta: metav1.TypeMeta{Kind: "Machine"},
+					ObjectMeta: metav1.ObjectMeta{
+						Annotations:       make(map[string]string),
+						Name:              "machine",
+						Namespace:         namespace,
+						Labels:            map[string]string{"foo": "bar"},
+						OwnerReferences:   []metav1.OwnerReference{{Kind: "MachineSet"}},
+						CreationTimestamp: metav1.Time{Time: time.Now().AddDate(0, 0, -1)},
```

Comment:
This isn't immediately obvious what this does, does it make it 1 day ago? 
```suggestion
						CreationTimestamp: metav1.Time{Time: time.Now().AddDate(0, 0, -1)}, // Now minus one day
```

Reply:
yes, I'll add the comment